thirdparty/portmidi/pm_win/pmwinmm.c

```diff
@@ -40,14 +42,14 @@
 
 /* callback routines */
 static void CALLBACK winmm_in_callback(HMIDIIN hMidiIn,
-                                       WORD wMsg, DWORD dwInstance, 
-                                       DWORD dwParam1, DWORD dwParam2);
+										WORD wMsg, DWORD_PTR dwInstance,
+										DWORD_PTR dwParam1, DWORD_PTR dwParam2);

```

Comment:
Here too please avoid the white space changes.

In general we indent by 6 spaces, not by tabs. Not in thirdparty code though, but there we don't change it either